### DISCUSSION TITLE
Future plan to be compatible with Wolfram SystemModeler

### DISCUSSION THREAD

**Discussion by lambtt** [lbl-srg/modelica-buildings#3251] [General]
Hi, would there be any plan to make the MBL compatible with Wolfram SystemModeler?

I used that software to run some examples in MBL, some could work, while some could not.

Thanks!

**Comment by mwetter** [comment]
We currently don't have funding to work on coverage with Wolfram SystemModeler, but we test the library with Dymola, Optimica/IMPACT and OpenModelica. If you think there is any violation of the Modelica Language Standard in our models that should be fixed, then please report it.

**Reply by lambtt** [reply]
Thanks, Michael, it helps a lot!
